etc/lime-elements.api.md

```diff
@@ -1560,6 +1566,7 @@ namespace JSX_2 {
     }
     // @beta
     interface LimelTextEditor {
+        "menuItems"?: EditorButton[];

```

Comment:
The control here is quite limited, as it doesn't really say anything about the order in which you place the buttons.

We actually do have complete control over this when interfacing wit the basic menu module, but is it worth the effort? 🤔